lib/src/algorithms.dart

```diff
@@ -148,7 +167,7 @@ class PixelMatching extends DirectAlgorithm {
   /// (0.05 * 256) of another RGB value.
   var tolerance;
 
-  PixelMatching({double this.tolerance = 0.05});
+  PixelMatching({bool this.ignoreAlpha = true, this.tolerance = 0.05});
```

Comment:
should this be set as false by default since most people will not look into documentation also I feel like a majority of images that we see being compared are pngs.

Reply:
That's a good point. I'll add in the change